assets/scripts/app/routing.js

```diff
@@ -77,6 +77,14 @@ export function goGoogleSignIn () {
   })
 }
 
+export function goPasswordSignIn () {
```

Comment:
The call to `goPasswordSignIn` includes a locale parameter, but this does not receive it or do anything with it.